src/widgets/NftDetailWidget/subcomponents/NftDetailActivity/NftDetailActivity.scss

```diff
@@ -0,0 +1,17 @@
+@import "src/styles/index";
+
+.nft-detail-activity {
+	&__activity {
+		border: .1rem #696969 solid;
+		padding: 1rem;
+		display: flex;
+		margin-bottom: 1rem;
```

Comment:
https://github.com/airswap/airswap-marketplace/blob/main/CONTRIBUTING.md#order-of-properties